packages/baselayer/src/generators/eslint.ts

```
@@ -26,52 +23,111 @@ export function generateESLintConfig(config: OutfitterConfig): Array<ESLintConfi
+      rules: eslintOverrides.rules,
+    });
+  }
 
-    // Handle other ESLint config properties
-    if (overrides.eslint.ignores) {
-      baseConfig.unshift({
-        ignores: overrides.eslint.ignores,
-      });
-    }
+  // Handle any other ESLint config properties that might be added in the future
+  const { ignores: _ignores, rules: _rules, ...otherOverrides } = eslintOverrides;
```

ESLint

[error] '_ignores' is assigned a value but never used.

(@typescript-eslint/no-unused-vars)